Manual.md

```diff
@@ -1616,6 +1619,23 @@ The following variables influence how Haskell packages are built:
 - `make_build_args`: This is passed as-is to `stack build ...`, so
   you can add your `--flag ...` parameters there.
 
+<a id="pkgs_nodejs"></a>
+### Node.js packages
+
+Node.js packages should be built with the `nodejs` build style, if possible.
+The `nodejs` build style takes care of downloading yarn/npm dependencies
+and running builds via package.json scripts.
+
+The following template variables influence how Node.js packages are built:
+- `nodejs_build_script`: the package.json script that builds the app files
+	(default: `build`).
+- `nodejs_install_script`: app installing script, which most probably 
+	doesn't even exist and you should redefine `do_install`
+	in your build script. 
+- `nodejs_packager`: whether the build script should use yarn or npm.
+	Set to `yarn` if the `yarn.lock` file exists, else it's set to `npm`
+- `NODE_ENV`: set to `production` by default
```

Comment:
it should be exported in the build style, otherwise just setting it won't affect the build.